code/_core/obj/item/tape.dm

```diff
@@ -0,0 +1,58 @@
+/obj/item/tape
+	name = "tape"
+	desc = "Finally, some good fucking music."
+	desc_extended = "Old tape with a single pre-recorded song. Use it in hand to start playing it for yourself to hear, or turn the currently playing song off."
+	icon = 'icons/obj/item/tape.dmi'
+	icon_state = "inventory"
+	value = 1250
+	value_burgerbux = 1
+	rarity = RARITY_RARE
+	weight = 0.25
+	size = SIZE_1
+	var/song_to_play = null
+
+/obj/item/tape/click_self(var/mob/caller,var/atom/object)
+
+	INTERACT_CHECK
+	INTERACT_DELAY(20)
+
+	if(icon_state == "inventory")
+		play_sound(song_to_play,get_turf(src),loop=TRUE)
```

Comment:
Again the sound system can't really support moving music so this should only play to the caller and be disabled when the player removes it from their inventory.

Reply:
:pensive: Yeah it'll be unticked.

Reply:
I think.

Reply:
Oh wait what you've written actually works.